traffic_portal/test/integration/specs/Types.spec.ts

```diff
@@ -36,39 +36,50 @@ describe('Setup API for Types Test', () => {
 types.tests.forEach(async typesData => {
     typesData.logins.forEach(login => {
         describe(`Traffic Portal - Types - ${login.description}`, () => {
+            afterEach(async function () {
+                await typesPage.OpenTypesPage();
+            });
+            afterAll(async function () {
+                expect(await topNavigation.Logout()).toBe(true);
+            })
             it('can login', async () => {
                 browser.get(browser.params.baseUrl);
                 await loginPage.Login(login);
-                expect(await loginPage.CheckUserName(login)).toBeTruthy();
-            });
-            it('can open types page', async () => {
+                expect(await loginPage.CheckUserName(login)).toBe(true);
                 await typesPage.OpenConfigureMenu();
-                await typesPage.OpenTypesPage();
             });
-
+            typesData.toggle.forEach(toggle => {
+                it(toggle.description, async () => {
+                    if(toggle.description.includes('hide')){
+                        expect(await typesPage.ToggleTableColumn(toggle.Name)).toBe(true);
+                    }else{
+                        expect(await typesPage.ToggleTableColumn(toggle.Name)).toBe(true);
+                    }
```

Comment:
This looks like it's doing the same thing regardless of which path is taken - is that check necessary?

Reply:
Fixed the check. Apparently, the function check was not cap.